### DISCUSSION TITLE
QField for Fibre Field Survey

### DISCUSSION THREAD

**Discussion by Asuduq** [opengisch/QField#5669] [Q&A]
I work in a Fibre Telecom company as the GIS guy. I'm to build their spatial database from ground up, so I QField for field survey.
Here are my encounters:
I have couple of layers 'Handhole, Splitter Box, Cabinet, Closure etc'.

- Each layer has it own Tag ID(i.e for Handhole: CV-HH-R1-0001), how can I autogenerate this ID serially, that is after point 0001 is done next point gets CV-HH-R1-0002
- I wish to have a table where if either a Splitter Box or Closure is in a Handhole, it keeps the record store in individual layer. For instance, I checked the box the a Closure in within a Handhole, the Tag ID and other records surveyed also goes to the Closure layer.

Thank you

**Comment by timbo-barnes** [comment]
This is more a general QGIS question so would likely be better answered somewhere else.
